sdk/resourcemanager/azure-resourcemanager-cosmos/pom.xml

```diff
@@ -146,6 +146,7 @@
         <artifactId>revapi-maven-plugin</artifactId>
         <version>0.11.2</version> <!-- {x-version-update;org.revapi:revapi-maven-plugin;external_dependency} -->
         <configuration>
+          <skip>true</skip>
```

Comment:
what breaking change do we have so that we need to add skip flag?

Reply:
non-final class to final class due to remove of JsonFlatten